packages/adapters/src/index.ts

```diff
@@ -10,3 +10,6 @@ export * from "./docker-networking";
 
 export * from "./kubernetes-sequence-adapter";
 export * from "./kubernetes-instance-adapter";
+
+export * from "./process-sequence-adapter";
```

Comment:
We should consider some tree-shaking here at some point. k8s adapter is making STH quite heavy.